.rubocop.yml

```diff
@@ -61,6 +61,11 @@ Style/CollectionMethods:
 Style/WordArray:
   Enabled: false
 
+# in rspec, expect{...} is conventional
```

Comment:
That doesn't match my expectation (so to speak), @atz. I would expect to see `expect {...}` vs. `expect{...}`.